views/layout.php

```diff
@@ -73,9 +73,10 @@
         
 <div class="col-sm-2">
     <div id="sideWidgetBar">
-    <img style="height: 50px; width: 50px;" src="views/images/twitterIcon.png" alt="placeholder social media icon" hspace="5"/>
-    <img style="height: 50px; width: 50px;" src="views/images/youtubeIcon.png" alt="placeholder social media icon" hspace="5"/>
-    <img style="height: 50px; width: 50px;" src="views/images/instagramIcon.png" alt="placeholder social media icon" hspace="5"/>
+    <img style="height: 50px; width: 50px;" src="views/images/twitterIcon.png" alt="placeholder social media icon" hspace="5"/><p class="tname"><a href="https://twitter.com/skyGITwomen/">@skyGITwomen</a></p>  <div class="tweet-container"><div class="tweet">
```

Comment:
It would be good if we could turn the images into links. We can look at doing this when we meet up on Tuesday.